### PR TITLE
Add intro to readme

### PR DESCRIPTION
**Pull request by StackScribe**

### CODE REVIEW

File: walkthrough/readme.md [hunk 1]
```diff
@@ -1,18 +1,51 @@
...
+Keptn and OpenFeature
+to gain better observability and management
+of your cloud-native software deployment.
+
+After some initial setup,
+you will do the following:
+
+1. Create and deploy a simple application using ArgoCD.
+1. Install OpenTelemetry into your application for some basic observability,
+   then install Keptn and see the expanded observability it provides.
```
Comment (agardnerIT) [inline walkthrough/readme.md:15]: then install Keptn and for the expanded observability of deployments and DORA metrics that it provides...

**Review by agardnerIT — CHANGES_REQUESTED** [review]
Once small things in comments are addressed, this is good to be merged.

File: walkthrough/readme.md [hunk 2]
```diff
@@ -1,18 +1,51 @@
...
+This masterclass guides illustrates how to use
+Keptn and OpenFeature
+to gain better observability and management
+of your cloud-native software deployment.
+
+After some initial setup,
+you will do the following:
+
+1. Create and deploy a simple application using ArgoCD.
+1. Install OpenTelemetry into your application for some basic observability,
```
[the remark this answers is not in the document]
Comment (StackScribe) [reply walkthrough/readme.md:14]: done

File: walkthrough/readme.md (hunk 1 above)
Comment (StackScribe) [reply walkthrough/readme.md:15]: Done -- I think.  You and "and for the expanded observability" so I took a crack at it.

**Review by agardnerIT — APPROVED** [review]

**Pull request merged**